src/install.rs

```diff
@@ -52,12 +54,23 @@ pub fn install(
     if let InstanceType::AppImage = instance_type {
         fs::rename(&archive_file, &executable_path)?;
     } else {
-        archive::unpack(&archive_file, &destination, true)?;
+        if !cfg!(target_os = "macos") {
+			archive::unpack(&archive_file, &destination, true)?;
```

Comment:
Mixed tabs / spaces. This should be easily fixed by running `cargo fmt`.

Reply:
Ran the command. I am using Visual Studio Code, didn't know it was messing this up.